contracts/multisig-ism-metadata/src/main.sw

```diff
@@ -1,25 +1,12 @@
 library;
 
-use std::
```

Comment:
This is super annoying, the new version of forc fmt always wants to add these newlines here